src/components/ErrorBoundary.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import Alert from "@material-ui/lab/Alert";
+
+function defaultFormatter(error) {
+  return (
+    <Alert severity="error">
+      Yikes! Something went wrong loading the OSCAL data. Sorry, we&apos;ll look
+      into it. ({error.message})
+    </Alert>
+  );
+}
+
+class ErrorBoundary extends React.Component {
+  constructor(props) {
+    super(props);
+    this.errorFormatter = props.errorFormatter ?? defaultFormatter;
+    this.state = { error: undefined };
+  }
+
+  componentDidCatch(error) {
+    this.setState({ error });
+  }
+
+  render() {
+    if (this.state.error !== undefined) {
+      return this.errorFormatter(this.state.error);
+    }
+    return this.props.children;
```

Comment:
```suggestion
    return this.props.children || null;
```
The reason why is that `this.props` or `this.props.children` can be undefined, and I was receiving the error:

```
Uncaught Error: ErrorBoundary(...): Nothing was returned from render. This usually means a return statement is missing. Or, to render nothing, return null.
```

Reply:
Good catch!